pipeline.py

```diff
@@ -0,0 +1,60 @@
+# type: ignore
+# pylint: disable=no-value-for-parameter,import-outside-toplevel,import-error,no-member
+from typing import List, Literal, Optional
+import click
+from kfp import dsl, compiler
+from kfp.kubernetes import use_config_map_as_env, use_secret_as_env
+
+K8S_NAME = "kfp-model-server"
+MOCKED_STAGES = ['sdg', 'train', 'eval']
+
+def pipeline_wrapper(mock: List[Optional[Literal[MOCKED_STAGES]]]):
```

Comment:
```suggestion
def pipeline_wrapper(mock: List[Literal[MOCKED_STAGES]]):
```
`TypeError: unhashable type: 'list'`

Reply:
What do you do to get that error? Which python version are you running? I don't see this. It's optional - we need to be able to NOT skip any stage.

Reply:
![image](https://github.com/user-attachments/assets/e62bb40f-131a-4df0-9bba-c57fb7b07cf1)